config.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
   enabled: true,
   workingDirectory: process.cwd(),
 
+  // The root directory containing the package.json for your project
+  // inside your git repository. This should be provided if your deployed
+  // application appears as a subdirectory of your git repository.
+  gitRoot: undefined,
```

Comment:
The explanation of what this is wasn't obvious upon reading the comment. Is this the absolute path of my git directory? Is it the absolute path of the directory containing the `package.json`? Don't you mean to ask for the relative path of the package.json in the source-repository. Also, we don't have a dependency on the source repository being git specifically – there is no reason, in principle why it has to be git.

Also, we probably need a longer explanation on why this value should be provided. Whether we do it in this comment could be debatable.

A better name for this variable would be something like 'appPathRelativeToRepository.